In [21]:
import pandas as pd
import numpy as np
from scipy import stats

TeleCall uses 4 centers around the globe to process customer order forms. They audit a certain % of the customer order forms. Any error in order form renders it defective and has to be reworked before processing. The manager wants to check whether the defective % varies by centre. Please analyze the data at 5% significance level and help the manager draw appropriate inferences

 Business Problem
Does the defective % varies significantly by centre ?

α == 0.05 (95% Confidence)

Y == Discrete
X == Discrete

Since there are more than 2 variable we will perform Chi-Square test

# Chi-Square Test

In [20]:
df=pd.read_csv('/content/Costomer+OrderForm.csv')


H0 == The defective % does not varies by centre

H1 == The defective % does varies by centre

In [24]:
stats.chi2_contingency([df['Phillippines'].value_counts(),df['Indonesia'].value_counts(),df['Malta'].value_counts(),df['India'].value_counts()])

(3.8589606858203545, 0.2771020991233144, 3, array([[271.75,  28.25],
        [271.75,  28.25],
        [271.75,  28.25],
        [271.75,  28.25]]))

P value of Chi-Square test == 0.277 > α

# H0 is accepted
The defective % does not varies significantly by centres